gitopsaddon/gitopsaddon_install.go

```
@@ -219,6 +219,11 @@ func (r *GitopsAddonReconciler) installViaEmbeddedManifests() error {
 	}
 	klog.Info("Successfully templated and applied openshift-gitops-operator")
 
+	if err := r.patchServiceAccountsWithImagePullSecrets(GitOpsOperatorNamespace); err != nil {
+		klog.Warningf("Failed to patch operator ServiceAccounts with imagePullSecrets: %v", err)
+	}
+	r.deletePodsWithImagePullIssuesInNamespace(GitOpsOperatorNamespace)
```

golangci-lint (2.12.2)

[error] Error return value of `r.deletePodsWithImagePullIssuesInNamespace` is not checked

(errcheck)